pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -165,7 +165,8 @@ private void preGenesisSubscription(final BigInteger lastProcessedBlockNumber) {
   }
 
   private SafeFuture<EthBlock.Block> getHead() {
-    return getLatestEth1BlockNumber()
+    return waitForEth1ProviderToBeInSync()
+        .thenCompose((__) -> getLatestEth1BlockNumber())
```

Comment:
nit: I think the brackets are unnecessary here:
```suggestion
        .thenCompose(__ -> getLatestEth1BlockNumber())
```